### ISSUE TITLE
2014 Primary Riverside test failure

### ISSUE THREAD

**Issue by charles-difazio** [openelections/openelections-data-ca#109]
Missing write-in candidates.  No precinct-level results, but [county-level totals](http://www.voteinfo.net/elections/20140603/eresults/20140603_WI_RESULTS.pdf)

```
E           AssertionError: Riverside failed
E           assert {('"Bo" Bogda...): 46473, ...} == {('"Bo" Bogdan...): 46473, ...}
E             Omitting 96 identical items, use -v to show
E             Left contains more items:
E             {('Conrad Melton', 'Riverside', 67.0, 'State Assembly'): 58,
E              ('Floyd Harvey', 'Riverside', 42.0, 'U.S. House'): 8,
E              ('Howard L. Katz', 'Riverside', 71.0, 'State Assembly'): 11,
E              ('John Farr', 'Riverside', 60.0, 'State Assembly'): 34,
E              ('Ken Park', 'Riverside', 60.0, 'State Assembly'): 144,
E              ('Nicholas Shestople', 'Riverside', 75.0, 'State Assembly'): 35,
E              ('Nickolas Wildstar', 'Riverside', '', 'Governor'): 2,
E              ('Oliver Unaka', 'Riverside', 60.0, 'State Assembly'): 118}
E             Use -v to get the full diff
```

**Comment by carbonphyber** [comment]
Fixed in #155

**Comment by warwickmm** [comment]
I believe this can be closed.
